WDAC-Policy-Wizard/app/src/EventLogRuleConfiguration.cs

```diff
@@ -142,7 +139,39 @@ private void AddButton_Click(object sender, EventArgs e)
         }
 
         /// <summary>
-        /// Populates the custom rules panel UI with the contents from the CiEvent list
+        /// Populates the custom rules panel with CiEvent on table row highlight event
+        /// </summary>
+        /// <param name="sender"></param>
+        /// <param name="e"></param>
+        private void RowSelectionChanged(object sender, EventArgs e)
+        {
+            // Try to cast sender as Datagrid. If successful, the user highlighted (arrow keys or mouse click) a new row
+            var grid = sender as DataGridView;
```

Comment:
@bowerscd-corp hope this makes you happy! Good suggestion, though. Much easier than try-catch casting or smth else